### ISSUE TITLE
Unable to use in Node.js

### ISSUE THREAD

**Issue by Coxxs** [CzBiX/disasm-web#2]
`test.mjs`: (copied from example)
```js
import {
  Const, Capstone, loadCapstone,
} from 'capstone-wasm'

await loadCapstone()

const capstone = new Capstone(Const.CS_ARCH_X86, Const.CS_MODE_32)
// the code can be bytes array or Uint8Array
const code = [0x55, 0x8b, 0xec, 0x83, 0xc4, 0x0c, 0xc3]
const insns = capstone.disasm(code, {
  address: 0x1000,
})

insns.forEach(insn => {
  console.log(insn.mnemonic, insn.opStr)
})
```

```
ReferenceError: __dirname is not defined in ES module scope
    at capstone-test/node_modules/capstone-wasm/dist/capstone-wasm.js:17:57
    at capstone-test/node_modules/capstone-wasm/dist/capstone-wasm.js:647:19)
    at capstone-test/test.mjs:5:7
    at ModuleJob.run (node:internal/modules/esm/module_job:198:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:409:24)
    at async loadESM (node:internal/process/esm_loader:85:5)
    at async handleMainPromise (node:internal/modules/run_main:61:12)

Node.js v18.4.0
```

**Comment by CzBiX** [comment]
It looks like this has been fixed in the last few days in [emscripten](https://github.com/emscripten-core/emscripten/pull/17915). I will try it later.

**Comment by CzBiX** [comment]
Encountered more issues than expected.😥
fixed in 9d0334e733c2c81116ddda32ad886554585dfbe5.
